iambic/config/wizard.py

```diff
@@ -802,6 +803,15 @@ def configuration_wizard_aws_account_add(self):  # noqa: C901
                 )
                 return
 
+        read_only = bool(
+            questionary.confirm(
+                "Do you want to restrict IambicSpokeRole to read-only?\n"
+                "This will limit IAMbic capability to import",
```

Comment:
Nit: 
This will also have an impact on Azure AD, Google Workspace, and Okta support within the wizard because our wizard requires the secret to be created as an AWS secrets manager secret.

Reply:
I this this should only be asked if it's the hub account because `self.config.aws.spoke_role_is_read_only` is global. That would also mean if it's not the hub account you'd need to set read_only by:

`read_only = self.config.aws.spoke_role_is_read_only`

Reply:
> Nit:
> This will also have an impact on Azure AD, Google Workspace, and Okta support within the wizard because our wizard requires the secret to be created as an AWS secrets manager secret.

The naming may become controversial. Despite saying read_only (to iam and sso API), we did leave the read + write on prefix on iambic_config_secrets_*, just to at least iambic setup to work